### ISSUE TITLE
the table lines is not aligned when displaying Chinese character

### ISSUE THREAD

**Issue by l1t1** [duckdb/duckdb#2546]
```
D select 'DB2和PostgreSQL';
┌────────────────┐
│ DB2和PostgreSQL │
├────────────────┤
│ DB2和PostgreSQL │
└────────────────┘
Run Time: real 0.007 user 0.000000 sys 0.000000
D .mode table
D select 'DB2和PostgreSQL数据库';
+-------------------+
| DB2和PostgreSQL数据库 |
+-------------------+
| DB2和PostgreSQL数据库 |
+-------------------+
Run Time: real 0.007 user 0.000000 sys 0.000000
```
clickhouse can display lines correct
```
Connected to ClickHouse server version 21.10.2 revision 54449.

┌─'DB2和PostgreSQL数据库'─┐
│ DB2和PostgreSQL数据库   │
└─────────────────────────┘

1 rows in set. Elapsed: 0.213 sec.
```

**Comment by l1t1** [comment]
if you add length of the STRING  by the number of Chinese characters.  then all OK

**Comment by omo** [comment]
Unsolicited triage bot here: It seems that this bug can be closed thanks to #4662.
